accuracy issue with test_svd_compressed on armv7

(Platform info: ARMv7 RPi2, Python 2.7, NumPy 1.9.2 compiled against OpenBLAS)

I'm seeing a unittest failure on the RPi2 like this:
```
================================================================================================================== FAILURES ==================================================================================================================
____________________________________________________________________________________________________________ test_svd_compressed _____________________________________________________________________________________________________________

    def test_svd_compressed():
        m, n = 500, 250
        r = 10
        np.random.seed(4321)
        mat1 = np.random.randn(m, r)
        mat2 = np.random.randn(r, n)
        mat = mat1.dot(mat2)
        data = da.from_array(mat, chunks=(50, 50))

        u, s, vt = svd_compressed(data, r, seed=4321)
        u, s, vt = da.compute(u, s, vt)

        usvt = np.dot(u, np.dot(np.diag(s), vt))

        tol = 0.2
        assert np.allclose(np.linalg.norm(mat - usvt),
                           np.linalg.norm(mat),
                           rtol=tol, atol=tol)  # average accuracy check

        u = u[:, :r]
        s = s[:r]
        vt = vt[:r, :]

        s_exact = np.linalg.svd(mat)[1]
        s_exact = s_exact[:r]

        assert np.allclose(np.eye(r, r), np.dot(u.T, u))  # u must be orthonormal
        assert np.allclose(np.eye(r, r), np.dot(vt, vt.T))  # v must be orthonormal
>       assert np.allclose(s, s_exact)  # s must contain the singular values
E       assert <function allclose at 0x761a4830>(array([ 422.22018238,  407.01168049,  392.40207352,  382.40705943,\n        351...4.51834555,  337.39779492,  302.80309518,\n        298.75930902,  274.74088453]), array([ 423.22257407,  408.0676158 ,  393.58177299,  383.76407994,\n        354...7.06991178,  339.22500436,  307.06042952,\n        300.98938086,  276.74680298]))
E        +  where <function allclose at 0x761a4830> = np.allclose

../work/dask/array/tests/test_linalg.py:133: AssertionError
```
The singular values are close to the NumPy result, but not that close (<1% deviation).

--- a/dask/array/tests/test_linalg.py
+++ b/dask/array/tests/test_linalg.py
@@ -102,16 +102,17 @@ def test_linalg_consistent_names():
     assert same_keys(v1, v2)
 
 
+@pytest.mark.slow
 def test_svd_compressed():
-    m, n = 500, 250
+    m, n = 2000, 250
     r = 10
     np.random.seed(4321)
     mat1 = np.random.randn(m, r)
     mat2 = np.random.randn(r, n)
     mat = mat1.dot(mat2)
-    data = da.from_array(mat, chunks=(50, 50))
+    data = da.from_array(mat, chunks=(500, 50))
 
-    u, s, vt = svd_compressed(data, r, seed=4321)
+    u, s, vt = svd_compressed(data, r, seed=4321, n_power_iter=2)
     u, s, vt = da.compute(u, s, vt)
 
     usvt = np.dot(u, np.dot(np.diag(s), vt))

